support-frontend/assets/helpers/productPrice/productOptions.ts

```diff
@@ -72,6 +72,11 @@ const paperProductsWithoutDigital = {
 	EverydayPlus: Everyday,
 } as Record<ProductOptions, ProductOptions>;
 
+const correspondingPaperProducts = {
```

Comment:
Would `allPaperProducts` be a better name for this?

Reply:
Possibly? Its purpose is to look up the product option with/without digital that corresponds to the one selected, but I don't feel great about the name.

Reply:
I had a bit of a think about this and decided that a function was both clearer and easier to name than an object, and added a comment with it for additional clarity.